Rendering/Shader/OffGeometryEffect.be

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<effect>
+  <pipelinestate name="main">
+    <D3D11>
+      <VertexShader filename="D3D/OffShader.hlsl" entry="VS_main"/>
+      <GeometryShader filename="D3D/OffShader.hlsl" entry="GS_main"/>
+      <PixelShader filename="D3D/OffShader.hlsl" entry="PS_main"/>
+    </D3D11>
+    <D3D12>
+      <VertexShader filename="D3D/OffShader.hlsl" entry="VS_main"/>
+      <GeometryShader filename="D3D/OffShader.hlsl" entry="GS_main"/>
```

Comment:
Still, no geometry shader.

Reply:
I recommend a joint session to discuss shaders and correct coding practice. Please get in touch and clear this out.

Reply:
Resolved: 5f612a2